src/main/scala/com/github/jberggg/router/RouterDsl.scala

```diff
@@ -0,0 +1,33 @@
+package com.github.jberggg.router
+
+import cats.implicits._
```

Comment:
This is the recommended import in modern Cats :)

```suggestion
import cats.syntax.all._
```

Reply:
Ahw, I wasn't aware of that 👍🏽